### DISCUSSION TITLE
I wanted to install the pgsql extension in ppostgresql 15, but it was an error

### DISCUSSION THREAD

**Discussion by Jimlisty** [rdkit/rdkit#6552] [FAQ]
(rdkit) D:\rdkit-master\Code\PgSQL\rdkit\build>cmake --build . --parallel %NUMBER_OF_PROCESSORS% --target install --config Release
MSBuild version 17.6.3+07e294721 for .NET Framework

  Checking Build System
  Building Custom Rule D:/rdkit-master/Code/PgSQL/rdkit/CMakeLists.txt
  adapter.cpp
D:\rdkit-master\Code\PgSQL\rdkit\adapter.cpp(49,10): fatal  error C1083: Unable to open the include file: “GraphMol/RDKitBase.h”: No such file
 or directory [D:\rdkit-master\Code\PgSQL\rdkit\build\rdkit.vcxproj]

I see that there is no GraphMol folder in this folder, I try to copy it in the parent directory, and still get an error. The pre-set cmake has been set and built, I don't know how to solve it next

**Comment by ptosco** [comment]
You will need to build the whole RDKit source tree from the root directory, not just the `Code\PgSQL\rdkit` subdirectory; i.e., `cd` into `D:\rdkit-master` and `mkdir build & cd build` there.
You'll need to set some PostgreSQL-related CMake switches to `ON`; see [this documentation](https://github.com/rdkit/rdkit/tree/master/Code/PgSQL/rdkit#cartridge-installation-into-non-conda-postgresql-1) for detailed instructions how to build the PostgreSQL cartridge on Windows.
